test_conformance/integer_ops/test_abs.c

```diff
@@ -324,6 +326,10 @@ int test_integer_abs(cl_device_id device, cl_context context, cl_command_queue q
         log_info("Failed on %d types\n", fail_count);
         return -1;
     }
+    if(compile_count) {
+        log_info("Failed to compile %d kernels\n", compile_count);
```

Comment:
This will print a confusing message when running the tests in source generation mode.

Reply:
ok, so this relates to adding a dedicated macro symbolic name as you suggested above, right?